src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,6 +66,11 @@ public function getDataFormSpecifications()
         return $tests;
     }
 
+    public function testLineBreaksBeforeDocumentSeparatorAndComment()
+    {
+        $this->parser->parse(file_get_contents(__DIR__.'/Fixtures/LineBreaksBeforeDocumentSeparatorAndComment.yml'));
```

Comment:
I would prefer to inline the YAML string instead of loading a file here (just like we do in the other tests).